src/components/v2/V2Project/V2ProjectReconfigureModal/index.tsx

```diff
@@ -272,32 +395,33 @@ export default function V2ProjectReconfigureModal({
         },
         onConfirmed() {
           setReconfigureTxLoading(false)
-          onOk()
+          exit()
         },
       },
     )
+
+    if (!txSuccessful) {
+      setReconfigureTxLoading(false)
+    }
   }, [
     editingFundAccessConstraints,
     editingFundingCycleMetadata,
     editingFundingCycleData,
     reconfigureV2FundingCycleTx,
     editingPayoutGroupedSplits,
     editingReservedTokensGroupedSplits,
-    onOk,
+    exit,
   ])
 
   const saveFundingTab = () => {
-    setFundingHasSavedChanges(true)
```

Comment:
**Question**: why are we getting rid of these? Are they not needed for your recent 'you have unsaved changes' PR?